src/v3/src/components/IdentifierContainer/IdentifierContainer.tsx

```diff
@@ -33,47 +31,32 @@ const IdentifierContainer: UISchemaElementComponent<{
       alignItems="center"
       marginBlockEnd={4}
       maxWidth={1}
-      className={mainContainerClasses}
+      className="no-translate"
```

Comment:
Can we switch this to `translate="no"` since that should be possible with a MUI `Box` and then remove the `className`

Reply:
That's fair, so you already tested and removing this `.no-translate` will break english leak CI? I might have a potential fix for the Preact issue coming

Reply:
Actually for now english leaks tests are being run in CI only for v2, not for v3 
I see test failures when running manually `yarn test:testcafe-run-en-leaks` (with `yarn workspace v3 dev`)
So I think it's fine to remove classes `no-translate` already and probably enable english leaks tests for v3 in the future